google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -247,8 +281,12 @@ def run(self):
         try:
             is_gcs_summary_enabled = (
                 inventory_notifier_config.get('gcs_summary').get('enabled'))
-            is_email_summary_enabled = (
-                inventory_notifier_config.get('email_summary').get('enabled'))
+            if self.notifier_config.get('email_connector'):
+                is_email_summary_enabled = True
```

Comment:
I am not sure why this `if` is needed.  Should `is_email_summary_enabled` really be set to true here by only checking for `email_connector`?

Reply:
I've added this `if` clause earlier to send inventory summary whenever email config is provided but I can remove this to have them explicitly enable/disable inventory summary.